kubernetes/annotations_and_labels.md

```diff
@@ -34,13 +31,29 @@ This page defines common annotations and labels we set in Kubernetes objects.
   services (i.e. K8s components) or `managed` for Giant Swarm managed services
   (i.e. networking, DNS, monitoring, ingress controller, etc.). Latter would be
   managed by `chart-operator`.
-- `release.giantswarm.io/version` - value should be the used Giant Swarm release
-  version, e.g. `2.3.0`.
-- `OPERATOR.giantswarm.io/version` - value should be the used operator version
-  bundle version, e.g. `kvm-operator.giantswarm.io/version=1.0.0`.
+- `release.giantswarm.io/version` - value should be Giant Swarm release
+  version, e.g. `release.giantswarm.io/version=2.3.0`.
+- `OPERATOR.giantswarm.io/version` - value should be an operator version, e.g.
+  `kvm-operator.giantswarm.io/version=1.0.0`. This is not the same as release
+  version. This value can be the same in multiple releases. This label is used
+  by the operator to recognize which object it should reconcile.
```

Comment:
Labels ending with `/version` are used for nodes. I completely forget how they are used. Let's make sure there is no interference or find out strategy to solve it.